vta/hardware/intel/Makefile

```diff
@@ -0,0 +1,82 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Directories
+ROOTDIR = $(CURDIR)
+BUILD_NAME = build
+BUILD_DIR = $(ROOTDIR)/../../$(BUILD_NAME)/hardware/intel
+SCRIPT_DIR = $(ROOTDIR)/scripts
+SRC_DIR = $(ROOTDIR)/../chisel
+TEST_DIR = $(ROOTDIR)/../../tests/hardware/common
+INCLUDE_DIR = $(ROOTDIR)/../../include
+
+# Executables
+TCLSH = quartus_sh
+
+# mode, options: skip_sim, sim
+MODE = skip_sim
+# Debug flag
+DEBUG = false
+# Prevent generation of DSP
+NO_DSP = false
+
+# Process VTA JSON config
+VTA_CONFIG = python $(CURDIR)/../../config/vta_config.py
+CFLAGS := $(shell ${VTA_CONFIG} --cflags)
+VTA_TARGET := $(shell ${VTA_CONFIG} --target)
+
+#---------------------
+# Compilation parameters
+#--------------------
+
+# Derive config name
+CONF = $(shell ${VTA_CONFIG} --cfg-str)
+IP_BUILD_PATH = $(BUILD_DIR)/chisel/$(CONF)
+HW_BUILD_PATH = $(BUILD_DIR)/quartus/$(CONF)
+
+# IP file path
+IP_PATH = $(BUILD_DIR)/chisel/$(CONF)/solution0/impl/ip/intel_fpga_com_chisel_vta_1_0.tgz

```

Comment:
This path is a product of the HLS toolchains we should simplify this path as it should just point to an output of the Chisel flow

Reply:
it should simply be `VTA.DefaultDe10Config.v`; the tar is not necessary unless expected by the Quartus toolchains

Reply:
This now simply points to `VTA.DefaultDe10Config.v`.